opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/StatusController.java

```diff
@@ -55,8 +55,8 @@ public Response getStatus(@PathParam("uuid") String uuid) {
             return Response.status(Response.Status.NOT_FOUND).build();
         }
 
-        if (apiTask.isCompleted()) {
-            return Response.status(apiTask.getResponseStatus()).build();
+        if (apiTask.isDone()) {
```

Comment:
I think this should be still using `isCompleted`, the same below

Reply:
`getResponse()` relies on `isDone()` and handles exceptions. `isCompleted()` would indicate only successful result of the task.